apps/aecore/src/aecore_sup.erl

```diff
@@ -60,6 +60,7 @@ init([]) ->
                                  ?CHILD(aec_metrics_rpt_dest, 5000, worker),
                                  ?CHILD(aec_keys, 5000, worker),
                                  ?CHILD(aec_tx_pool, 5000, worker),
+                                 ?CHILD(aec_block_generator, 5000, worker),
```

Comment:
Could you please update (or delete) representation of supervision tree in comment above in module?